mirgecom/simutil.py

```diff
@@ -93,8 +93,8 @@ def __init__(self, step, t, state):
 
 def sim_checkpoint(discr, visualizer, eos, q, vizname, exact_soln=None,
                    step=0, t=0, dt=0, cfl=1.0, nstatus=-1, nviz=-1, exittol=1e-16,
-                   constant_cfl=False, comm=None, viz_fields=None, overwrite=False,
-                   vis_timer=None):
+                   s0=None, kappa=None, constant_cfl=False, comm=None,
+                   viz_fields=None, overwrite=False, vis_timer=None):
```

Comment:
The `viz_fields` mechanism is the way to add user-defined quantities to the visualization dumps. [Here is an example](https://github.com/illinois-ceesd/mirgecom/blob/171e59fe483f11448b4d9967dd10a3512753bbcf/examples/autoignition-mpi.py#L228) of how to use it.

Reply:
https://github.com/illinois-ceesd/mirgecom/pull/213/commits/77b02639eb730b820de6dd70f405f58451b2e5fb removes changes to simutil.py and presents how to use vis_fields in doublemach-mpi.py